github/Requester.py

```diff
@@ -424,6 +424,9 @@ def __createException(self, status, headers, output):
             or output.get("message")
             .lower()
             .endswith("please wait a few minutes before you try again.")
+            or output.get("message")
```

Comment:
Maybe it is more intuitive to move that `startswith` up to the first `startswith` to collect `startswith` conditions and `endswith` conditions.